modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/EditFolderMVCRenderCommand.java

```diff
@@ -39,8 +40,14 @@ protected void checkPermissions(
 			PermissionChecker permissionChecker, Folder folder)
 		throws PortalException {
 
-		_folderModelResourcePermission.check(
-			permissionChecker, folder, ActionKeys.UPDATE);
+		if (!_folderModelResourcePermission.contains(
```

Comment:
we have to check at least UPDATE permission if the FF is disabled

Reply:
will take a look, thanks

Reply:
changed